packages/hooks/src/useKeyStroke.ts

```diff
@@ -8,6 +8,12 @@ export type KeyOptions = {
   eventName?: KeyStrokeEventName
   ref?: (() => EventTarget | null) | null
   passive?: boolean
+  /**
+   * Whether to include key repeat events
+   * @default false
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/repeat
+   */
+  includeRepeat?: boolean
```

Comment:
rename to `repeat`, it'll be more intuitive to people and is more aligned with the KeyboardEvent spec.